### ISSUE TITLE
Question for training of Burst De-noising.(loss function, data generation and training tricks)

### ISSUE THREAD

**Issue by hanhhhhh** [akshaydudhane16/Burstormer#7]
Hello,

We noticed that you mentioned in the article that all training utilized L1 Loss. However, in the Burst Denoising section, you referred to the experimental setups of KPN and BPN, where KPN employed the Basic Loss (L2 Loss + gradient loss). We are curious about the Loss function employed in your Burst Denoising.

Furthermore, in Burst Denoising, when converting Open Image data, apart from what was mentioned in KPN, are there any additional steps or specific considerations? During the training phase of Burst Denoising, were there any other training techniques applied? Currently, we are unable to replicate the training accuracy. 

We eagerly await your response!

**Comment by cvkaiming** [comment]
> 

Hi, did you get the results you got from the BurstSR Track2 part of the training up to the PSNR in the article, thanks!

**Comment by hanhhhhh** [comment]
Sorry, I've only tried the denoising part of the code so far.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "akshaydudhane16/Burstormer"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年8月9日(星期三) 上午9:09
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [akshaydudhane16/Burstormer] Question for training of Burst De-noising.(loss function, data generation and training tricks) (Issue #7)





   
Hi, did you get the results you got from the BurstSR Track2 part of the training up to the PSNR in the article, thanks!

**Comment by GopiRajuMatta** [comment]
Hello @akshaydudhane16!

Thank you for releasing the code for testing.

Can you also please update the training code for de-noising.

Thank you
Gopi

**Comment by Egkang-Luis** [comment]
@hanhhhhh 
Hi hanhhhhh. I think you success making training code for burst denoising task.

I would like to make training code for dataset, but I am curious something.

If you don't mind, could you share which dataset you download in open image dataset? Also, If you can share the code for dataset, it will be helpful to me.

My email address is eunggukang@gmail.com

Thanks.
Luis Kang.

**Comment by YohanJRoh** [comment]
@hanhhhhh 
@Egkang-Luis Egkang-Luis
@akshaydudhane16 
Dear all,

Could you please share your **training code** for Color burst denoising burstormer
as well as the **creation code for dataset**, etc?

My email address is yohan.roh@samsung.com

I would greatly appreciate it if you could share it with me. 

Thank you in advance.

Best regards and respect,
Yohan

**Comment by YoungP2001** [comment]
@akshaydudhane16 
@hanhhhhh 
@Egkang-Luis 
@YohanJRoh 

Dear all,

Could you share the processing scripts for denoising datasets? It would be even better if you could directly share the datasets.

My email is 15049114601@qq.com.

I would be very grateful for your help.

Best regards and respect,
Pei

**Comment by yyxmoon123** [comment]
@akshaydudhane16
@hanhhhhh
@Egkang-Luis
@YohanJRoh

各位，

能分享一下去噪数据集的处理脚本吗？如果能直接分享数据集就更好了。

我的邮箱是[yyx150232yyx@163.com](mailto:yyx150232yyx@163.com)。

我将十分感激您的帮助。
